app/src/main/java/com/eveningoutpost/dexdrip/Models/Tomato.java

```diff
@@ -48,7 +48,7 @@ public static BridgeResponse decodeTomatoPacket(byte[] buffer, int len) {
         final BridgeResponse reply = new BridgeResponse();
         // Check time, probably need to start on sending
         long now = JoH.tsl();
-        if(now - s_lastReceiveTimestamp > 10*1000) {
+        if(now - s_lastReceiveTimestamp > 3*1000) {
             // We did not receive data in 10 seconds, moving to init state again
```

Comment:
Are we sure this will never take more than 3 seconds even in really bad RF situations?

Also the comment below still references 10 seconds, should be updated to something more generic

Reply:
I have fixed the comment.
The main reason that we need this change is that now (actually for some time now) there is a retry code after crc errors. That retry takes time after 10 seconds, and that might cause an issue, so I have changed that to be 3 seconds.
The distance between the packets should be ~100ms. So no reason to see things taking more than 3 seconds.